packages/desktop-client/src/components/mobile/budget/index.tsx

```diff
@@ -68,22 +67,6 @@ export function Budget() {
     }
 
     init();
-
-    const unlisten = listen('sync-event', event => {
```

Comment:
No need for this since this is already handles by the global sync-event listener